src/test/java/com/ncc/aif/NistTA3ExamplesAndValidationTest.java

```diff
@@ -21,17 +21,17 @@
     private static final boolean FORCE_DUMP = false;
 
     private static final String LDC_NS = "https://tac.nist.gov/tracks/SM-KBP/2018/LdcAnnotations#";
-    private static final String ONTOLOGY_NS = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/SeedlingOntology#";
-    private static final CharSource SEEDLING_ONTOLOGY = Resources.asCharSource(
-            Resources.getResource("com/ncc/aif/ontologies/SeedlingOntology"),
+    private static final String ONTOLOGY_NS = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/LDCOntology#";
+    private static final CharSource LDC_ONTOLOGY = Resources.asCharSource(
+            Resources.getResource("com/ncc/aif/ontologies/LDCOntology"),
             StandardCharsets.UTF_8);
     private static NistTA3TestUtils utils;
 
     @BeforeAll
     static void declutterLogging() {
         // prevent too much logging from obscuring the Turtle examples which will be printed
         ((Logger) org.slf4j.LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME)).setLevel(Level.INFO);
-        utils = new NistTA3TestUtils(LDC_NS, ValidateAIF.create(ImmutableSet.of(SEEDLING_ONTOLOGY),
+        utils = new NistTA3TestUtils(LDC_NS, ValidateAIF.create(ImmutableSet.of(LDC_ONTOLOGY),
```

Comment:
Same comment as before regarding `ValidateAIF.createForLDCOntology()` and the `CharSource`.

Reply:
Fixed